calibration_node:
  ros__parameters:

    number_of_images_to_calibrate: 35
    board_dim: [5, 7]
    square_size: 22.0 # mm

    subscribers:
      camera: /camera/raw_frame

    auto_capture:
      mode: True            # Capture an image every
      time_for_frame: 1.0   # sec
    
    #calibration_path: auto  # Specify absolute path instead
    calibration_path: /home/rvd/Desktop/xarm_dev/src/dependencies/ROS2-HAL-Allied-Vision-Alvium1800U/allied_vision_camera/calibration/
